frontend/public/components/terminal.jsx

```diff
@@ -120,6 +120,7 @@ Terminal.propTypes = {
 
 Terminal.defaultProps = {
   padding: 30,
+  fullscreenPadding: 10,
```

Comment:
How does it look if set to 0?

Reply:
OK clearly 10 is better. I wonder if we should just hard-code the fullscreen padding. I don't see a lot of value making it a prop

Reply:
It's not a css padding, it's padding used to calc the width of the terminal.

Reply:
Right, I understand. But why does the fullscreen value need to be configurable? Any reason not to always use 10?

Reply:
I'm suggesting always using const value `10` in fullscreen mode (and only fullscreen mode, not changing padding)

Reply:
oh, umm, not sure.  was just following how `padding` was defined, which seems to indicate app dev could pass in padding as a prop.

Reply:
Ok, hardcoded the value, thanks